From 84a0d64c068f65dd073e13f6fcf55d11ed8a21a7 Mon Sep 17 00:00:00 2001
From: Peng Tao <tao.peng@emc.com>
Date: Wed, 22 Aug 2012 16:55:22 +0800
Subject: [PATCH 11/13] LU-1337 kernel: v3.5 defines INVALID_UID

With kernel commit 7a4e7408, Lustre doesn't need to redefine
INVALID_UID.

Signed-off-by: Peng Tao <tao.peng@emc.com>
Change-Id: I96b854cc51db735d8c985528c879fbeb5b049ab9
---
 lustre/include/lustre_cfg.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lustre/include/lustre_cfg.h b/lustre/include/lustre_cfg.h
index 7262be3..9d0fd52 100644
--- a/lustre/include/lustre_cfg.h
+++ b/lustre/include/lustre_cfg.h
@@ -286,7 +286,9 @@ static inline int lustre_cfg_sanity_check(void *buf, int len)
 
 #include <lustre/lustre_user.h>
 
+#ifndef INVALID_UID
 #define INVALID_UID     (-1)
+#endif
 
 /** @} cfg */
 
-- 
1.7.12

